### PR TITLE
DBZ-7570 docs update for use.nongraceful.disconnect config

### PR DESCRIPTION
**Pull request by acristu**
Related PRs:
https://github.com/osheroff/mysql-binlog-connector-java/pull/142
https://github.com/debezium/debezium/pull/5517

### CODE REVIEW

**Comment by jpechane** [comment]
@roldanbob Could you please check the wording and merge if it is OK? Thanks

File: documentation/modules/ROOT/pages/connectors/mysql.adoc [hunk 1]
```diff
@@ -3437,6 +3437,10 @@ The following table describes xref:mysql-advanced-connector-configuration-proper
 |`true`
 |A Boolean value that specifies whether a separate thread should be used to ensure that the connection to the MySQL server/cluster is kept alive.
 
+|[[mysql-property-use-nongraceful-disconnect]]<<mysql-property-use-nongraceful-disconnect, `+use.nongraceful.disconnect+`>>
+|`false`
+|A Boolean value that specifies whether the binlog client keepalive thread should use SO_LINGER with 0 timeout when terminating a stale connection. The goal is to avoid potential deadlock in `SSLSocketImpl.close`, see https://github.com/osheroff/mysql-binlog-connector-java/issues/133 for more details.
```
Comment (roldanbob) [inline documentation/modules/ROOT/pages/connectors/mysql.adoc:3442]:
```suggestion
|A Boolean value that specifies whether the binary log client's keepalive thread sets the `SO_LINGER` socket option to  `0` to immediately close stale TCP connections. 
Set the value to `true` if the connector experiences deadlocks in `SSLSocketImpl.close`.
ifdef::community[]
For more information, see https://github.com/osheroff/mysql-binlog-connector-java/issues/133 .
endif::community[]
```

**Review by roldanbob — CHANGES_REQUESTED** [review]
@jpechane @acristu @Naros Please take a look and let me know if these changes are ok. 
I conditionalized the external link for upstream use only, bc it seemed like TMI for the product version. Do you concur?

**Comment by Naros** [comment]
Hi @roldanbob we most definitely should conditionalize the link to the binlog client repository for downstream builds, so I do concur with the suggestion :+1:

**Comment by jpechane** [comment]
Closing, changes incorporated via https://github.com/debezium/debezium/pull/5529

Thanks!

**Pull request closed without merging**